csv2wiki/__init__.py

```diff
@@ -805,6 +787,42 @@ def _do_skel(self, skel, page_title, row, row_num):
             + " " + ("=" * skel.level)                             \
             + text
 
+    def _wikiize_cell(self, page_title, data_row, col_idx):
+        """Update a CELL to be ready for the wiki.
+
+        Data from the spreadsheet may not be properly ready to land
+        on mediawiki.  This is a central place to handle preparing
+        and updating those strings.
+
+        This is MediaWiki-specific, but I imagine a future version of
+        this might want to override with formatting for other wikis."""
+
+        cell = data_row[col_idx]
+
+        if cell.lower() == "null" and not self._null_as_value:
+            cell = ""
+        else:
+            # Mediawiki doesn't do tbody
+            cell = cell.replace("<tbody>", "").replace("</tbody>", "")
+            # Make soup
+            warnings.filterwarnings(
+                "ignore", category=UserWarning, module='bs4')
+            soup = BeautifulSoup(cell, "html.parser")
+            soup = wikify_anchors(soup)
+
+            cell = str(soup)
+
+            if col_idx == self._cat_col:
```

Comment:
Note that because categories are going to be coming in now as (possibly) comma-separated lists, the processing here is going to change soon.  It might be good to have a comment to that effect.

In the old data, there was one category per proposal, but now it's more like tags: there can be several categories -- I believe the "Sustainable Development Goals" column is this, basically.

So, thinking ahead... we might need a `cat_col_separator` parameter in the config file too, which, if present, tells us how to divide up the category column to generate multiple categories :-).

Reply:
I'm definitely waiting for that problem definition to be fully realized.